static/elements/chromedash-featurelist.js

```diff
@@ -388,24 +383,26 @@ class ChromedashFeaturelist extends LitElement {
   }
 
   render() {
```

Comment:
Nit: I believe it's not a good practice to have computation and new variables inside the render function. Move it out if possible.
Not blocking this PR. Maybe add a TODO.

Reply:
Added TODO.